packages/components/src/components/action-bar/action-bar.tsx

```diff
@@ -68,8 +71,33 @@ export class ActionBar extends LitElement {
     const groupCount =
       this.hasActionsEnd || !expandDisabled ? actionGroups.length + 1 : actionGroups.length;
 
+    let bufferSize = groupCount;
+    const actionBarContainerStyle = getComputedStyle(this.containerRef.value);
+
+    if (layout === "horizontal") {
```

Comment:
To confirm, `layout=vertical` doesn't need any special handling?

Reply:
Actually, I'll need to do more than just removing the conditional.  I have an idea I'll try.

Reply:
Ok, I just pushed an update to apply the same computation logic to both layout dimensions, so vertical layout is responding more in kind to the way horizontal works and how it previously behaved now.  I also added the action group 1px border into the calculation also.

@jcfranco Let me know if the way I wrote it is readable.  I opted to use ternaries inside each `getStylePixelValue()` call to grab the appropriate `inline` or `block` padding value.